systemd/crc-ng-env.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+if [ -n "$CRC_NG" ] || [ -n "$CRC_CLOUD" ]; then
+    exit 1
```

Comment:
shouldn't this be `exit 0`? in case there's a value set for `CRC_NG` or `CRC_CLOUD` we want to `exit 0` (so that self sufficient services run) and in case this env variables are not set `exit 1`